CONTRIBUTING.rst

```diff
@@ -225,6 +226,9 @@ spelling or other errors on the website/docs/code.
         confidential information like API tokens and authentication
         credentials.
 
+    E) If your issue is tagged with `Needs Test Case` please provide a code sample
+       so we can verify when the issue is resolved.
```

Comment:
need tests to avoid regression

Reply:
Test cases are not unit or functional tests.
Test cases are simply the most minimal programs possible that will reproduce a problem.
Most of the time we can create integration tests from test cases.

Reply:
Ah, I see.
I'll make this clear in the docs. Thanks for clarifying :)